example_secrets.tfvars

```diff
@@ -0,0 +1,9 @@
+# fake example secret data
+
+# rename this file to secrets.tfvars
+# never commit secrets to source control
+# include secrets during the run by using terraform plan -var-file secrets.tfvars.
+# See: https://www.terraform.io/intro/getting-started/variables.html
+aws_access_key_id = "YOURKEYHERE"
+aws_secret_access_key = "YOURSECRETACCESSKEYHERE"
+aws_key_pair_name = "KEYPAIRNAMEHERE"
```

Comment:
Why is this line needed? We don't have a variable named this in the scripts today.